instana/autoprofile/runtime.py

```diff
@@ -3,6 +3,7 @@
 
 import sys
 import signal
+import os
```

Comment:
I added the import because I am going also a bit through the files and checking the complaints in pycharm. It is complaining of unresolved import if I take it out because of line 42. `os.kill(os.getpid(), signum)`